core/flyout_base.ts

```diff
@@ -136,14 +137,14 @@ export abstract class Flyout extends DeleteArea implements IFlyout {
    * Function that will be registered as a change listener on the workspace
    * to reflow when blocks in the flyout workspace change.
    */
-  private reflowWrapper: Function | null = null;
+  private reflowWrapper: ((e: AbstractEvent) => void) | null = null;
```

Comment:
Hmm.  Technically this (and the below) is a breaking change, but I imagine that passing in a function that does not accept an `AbstractEvent` as its first parameter did not actually work previously, despite the sloppy signature.  Is that true?

Reply:
If you passed in a `() => void` that's fine because it's a subtype of `(anyParameters) => void`. If you passedin `(someTypeBesidesAnEvent) => void` then yeah that didn't work before. So this shouldn't break anything.

Reply:
Great.